packages/components/src/Listing.tsx

```diff
@@ -0,0 +1,48 @@
+import MonacoEditor, { useMonaco } from "@monaco-editor/react";
+import { editor } from "monaco-editor";
+import { useEffect } from "react";
+import { SubstanceConfig } from "./editing/languages/SubstanceConfig";
+
+export const monacoOptions: editor.IStandaloneEditorConstructionOptions = {
+  automaticLayout: true,
+  readOnly: true,
+  minimap: { enabled: false },
+  wordWrap: "on",
+  lineNumbers: "off",
+  fontSize: 16,
+  scrollbar: {
+    handleMouseWheel: false,
+  },
+  scrollBeyondLastLine: false,
+  renderLineHighlight: "none",
+};
+
+const Listing = ({
+  value,
+  width,
+  height,
+}: {
+  value: string;
+  width: string;
+  height: string;
```

Comment:
Weird that this component is not taking in an `Env`. Don't we need that for syntax highlighting? In fact, I'm not seeing any highlighting in all examples.

Reply:
fixed